### ISSUE TITLE
Filed Issues a11y android

### ISSUE THREAD

**Issue by benhammondmusic** [SatcherInstitute/health-equity-tracker#1061]
Some of these may be fixed; I cannot text on Android to confirm.

- [x] UC 03- Reading the What Is Health Equity > What Is Health Equity? page	Large Text	P2	Text is getting overlap with Display text is set to Large	"Environment:
Device: Pixel 3a xL
Android 11
Chrome 83.0.4103.106
Prerequisite: 
Ensure Display text size is set to large
Steps:
1. Launch the url in the chrome browser
2. Navigate to What Is Health Equity > What Is Health Equity? page
3. Observe the ¨Data¨ page is shown with text displayed overlap
Actual Result:
The text is getting overlap with Display text set to large
Expected Result:
The text should not get overlap with display text is set to large"

- [x] "US04 -Reading the About Us > The Project page
US12 -Reading Downloads and Methodology > Data Downloads page
US13-Reading Downloads and Methodology > Methodology page"	Large Text	P2	[A11y_HET] : Content is not rendering to large font setting_mWeb_1.15(c)	"Environment:
Device: Pixel 3 XL
Android version: 11
Chrome browser: 90.0.4430.210
Pre-requisites:
Turn on the Large font and keep the size as largest (Settings > Accessibility >Font size>Largest).
Steps to reproduce:
1.Go to https://healthequitytracker.org/, Login with valid credentials
2.Navigate to ‘The Project’,‘Data Downloads’ and ‘Methodology’ pages 
3.Validate the items on the page with increased font size.
Actual Result:
Content on the screen remains unchanged even when font size has been changed to largest.
Expected Result:
Items must scale accordingly when font size is changed to largest
Video: https://drive.google.com/file/d/1FuORv2ITm9pn372cA173YwmzecRIIov6/view?usp=sharing&resourcekey=0-27yGOT2CIYDAFQL2cWgh_A"

- [x] 02 Reading the home page	Screen reader		[a11y-HealthEquityTracker]”Skip to Main Content” link does not work with TalkBack_Android_1.4	"PREREQUISITE:
Google Pixel 3A
Android 11
Google Chrome 90.0.4430.210
1. Turn on TalkBack screen reader
STEPS TO REPRODUCE
1. Navigate to https://healthequitytracker.org/
2. Navigate to and activate the ""Expand Site Navigation"" button
3. Navigate to and activate the ""Explore the Data"" link
4. Navigate to the top of the page and activate the ""Skip to Main Content"" link
5. Observe that there is no screen reader announcement and focus does not move                                                                                                                                                                            EXPECTED RESULT:
Skip to Main Content” links are common and screen reader users will use them frequently to bypass information in order to reach the “main content. Screen readers will often announce that the link will take focus to another location on the current page as opposed to taking the user to a new page.
ACTUAL RESULT:
TalkBack announces the link but when the user activates the link, the screen reader focus does not move. The user focus remains on the link. No announcement is made that it is a “same page” style link.
